get_latest_configs.sh

```diff
@@ -0,0 +1,14 @@
+#bin/bash
+
+wget -q -O config.template.yaml https://raw.githubusercontent.com/calzoneman/sync/3.0/config.template.yaml
+mv config.template.yaml config.yaml
+echo -e "\e[32m Success! Config file downloaded from github and copied to config.yaml. Please update the config to match the settings specific to your application."
```

Comment:
Without checking the return code of `wget` you don't necessarily know it succeeded.  The easiest way to avoid this is to add `set -e` to the top of the script (below the `#!/bin/sh`) to crash the shell script if any command fails.

Reply:
Thanks for the tip! I will  use this in the future with other projects. This is not needed now since it will be copying the files directly from the sync folder which is not a git submodule.